js/content/common.js

```diff
@@ -1181,21 +1181,15 @@ let AugmentedSteam = (function() {
                 </div>
             </div>`);
 
-        let popup = document.querySelector("#es_popup");
-
         document.querySelector("#es_pulldown").addEventListener("click", () => {
             ExtensionLayer.runInPageContext(() => { ShowMenu("es_pulldown", "es_popup", "right", "bottom", true); });
         });
 
-        document.querySelector("#es_menu").addEventListener("click", function(e){
-            e.stopPropagation();
-        });
-
-        document.addEventListener("click", function(){
-            popup.classList.remove("open");
+        document.querySelector("#es_popup").addEventListener("click", () => {
+            ExtensionLayer.runInPageContext(() => { HideMenu("es_pulldown", "es_popup"); });
```

Comment:
Why are you hiding it here?
I don't see that behavior on Steam's menu

Reply:
This hides the menu if the link is opened in a new tab.

Reply:
Yes, but why?

Reply:
I don't quite understand the question, or should this be done in a different way? To put it more clearly, this fixes the popup still showing if the links are opened in a new tab (by having "Open external links in a new tab" checked). I suppose Steam's menu doesn't have this behaviour because none of them open in a new tab.